scripts/build_computecpp.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Copyright (C) 2022 Codeplay Software Limited
+# This work is licensed under the terms of the MIT license.
+# For a copy, see https://opensource.org/licenses/MIT.
+
+BUILD_DIR="build_sycl"
+
+rm -rf $BUILD_DIR
+mkdir $BUILD_DIR
+cd $BUILD_DIR || exit
+
+cmake ../ \
+	-DGLEW_LIBRARY=/usr/lib/x86_64-linux-gnu/libGLEW.so \
+	-DBUILD_SYCL=on \
+	-DUSE_COMPUTECPP=on \
+	-DComputeCpp_DIR=/home/joetodd/sources/ComputeCpp \
+	-DCOMPUTECPP_BITCODE="host-x86_64" \
+	-DSYCL_LANGUAGE_VERSION=202002\
```

Comment:
```suggestion
	-DSYCL_LANGUAGE_VERSION=2020\
```
There are newer versions than this, setting it to 2020 will get the newest of these IIRC